HST/organize_files.py

```
@@ -38,28 +39,45 @@ def organize_files_from_staging_to_bundles(proposal_id, logger):
     Inputs:
-        proposal_id    a proposal id.
+        staging_dir    path to an hst_<nnnnn> staging directory.
         logger         pdslogger to use; None for default EasyLogger.
     """
-    logger = logger or pdslogger.EasyLogger()
-    logger.info(f'Clean up staging directory for proposal id: {proposal_id}')
-    staging_dir = get_program_dir_path(proposal_id, None, root_dir='staging')
-    # Do nothing is the staging directory doesn't exist
     if not os.path.isdir(staging_dir):
         return
     for dir in os.listdir(staging_dir):
```

Ruff (0.15.14)

[error] Variable `dir` is shadowing a Python builtin

(A001)